roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -206,6 +226,37 @@
     - port: 9200
       targetPort: "restapi"
 
+- name: Set logging-prometheus service
+  oc_service:
+    state: present
+    name: "logging-prometheus"
```

Comment:
do we need `logging-prometheus-{{ es_component }}` for es and es-ops?  Same question below for the route

Reply:
I do not believe so.  I purposely left them out because I would imagine the same prometheus credentials and setup should be able to be used to scrape either ops or non ops Elasticsearch instance.  Adding one for each seems to just complicate the mater IMO.  I'm going with YAGNI for this one